core/controlplane/config/templates/cloud-config-controller

```diff
@@ -345,6 +345,23 @@ coreos:
 {{end}}
         Type={{.Experimental.EphemeralImageStorage.Filesystem}}
 {{end}}
+{{ if .SharedPersistentVolume }}
+    - name: load-efs-pv.service
+      command: start
+      content: |
+        [Unit]
+        Description=Load efs persistent volume mount
+        Wants=kube-node-taint-and-uncordon.service
+        After=kube-node-taint-and-uncordon.service
+        [Service]
+        Type=simple
+        RemainAfterExit=true
+        RestartSec=10
+        Restart=on-failure
+        ExecStartPre=/opt/bin/set-efs-pv
+        ExecStartPre=/usr/bin/systemctl is-active kube-node-taint-and-uncordon.service
+        ExecStart=/opt/bin/load-efs-pv
```

Comment:
Perhaps you're missing an entry for the `/opt/bin/load-efs-pv` script under `write_files:`?